### PR TITLE
add res2net backbone

### PR DESCRIPTION
**Pull request by ghostxsl**
add res2net backbone and models

### CODE REVIEW

File: configs/res2net/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Res2Net
```
Comment (qingqing01) [inline configs/res2net/README.md:1]: please also update  https://github.com/PaddlePaddle/PaddleDetection/blob/develop/docs/MODEL_ZOO_cn.md

File: ppdet/modeling/backbones/res2net.py [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+
+Res2Net_cfg = {
+    50: [3, 4, 6, 3],
+    101: [3, 4, 23, 3],
+    152: [3, 8, 36, 3],
+    200: [3, 12, 48, 3]
+}
+
+
+class ConvNormLayer(nn.Layer):
```
Comment (jerrywgz) [inline ppdet/modeling/backbones/res2net.py]: 能否复用resnet中的convnormlayer

File: configs/res2net/README.md (hunk 1 above)
Comment (ghostxsl) [reply configs/res2net/README.md:1]: done

File: ppdet/modeling/backbones/res2net.py (hunk 2 above)
Comment (ghostxsl) [reply ppdet/modeling/backbones/res2net.py]: done

**Review by qingqing01 — DISMISSED** [review]

**Review by qingqing01 — DISMISSED** [review]

**Pull request closed without merging**